apps/console/src/features/organization-discovery/components/add-organization-discovery-domains.tsx

```diff
@@ -172,17 +177,59 @@ const AddOrganizationDiscoveryDomains: FunctionComponent<AddOrganizationDiscover
     };
 
     /**
-     * Function to validate the input string is an email domain.
+     * Function to check whether an email domain is available.
+     *
+     * @param values - Email domains.
+     */
+    const checkEmailDomainAvailability = async (emailDomain: string): Promise<boolean> => {
```

Comment:
Let's check if we can add this method to feature's utils folder since this is used twice.